quic/s2n-quic-transport/src/connection/connection_impl.rs

```diff
@@ -1531,6 +1532,18 @@ impl<Config: endpoint::Config> connection::Trait for ConnectionImpl<Config> {
                 packet_interceptor,
             )?;
 
+            // try to process any post-handshake messages
+            let space_manager = &mut self.space_manager;
+            space_manager.post_handshake_crypto(
+                &mut self.path_manager,
+                &mut self.local_id_registry,
+                &mut self.limits,
+                datagram.timestamp,
+                &self.waker,
+                &mut publisher,
+                datagram_endpoint,
+            )?;
```

Comment:
This is only done on the client, right? It's probably a good idea to do something like
```suggestion
            if Config::ENDPOINT_TYPE.is_client() {
                // try to process any post-handshake messages
                let space_manager = &mut self.space_manager;
                space_manager.post_handshake_crypto(
                    &mut self.path_manager,
                    &mut self.local_id_registry,
                    &mut self.limits,
                    datagram.timestamp,
                    &self.waker,
                    &mut publisher,
                    datagram_endpoint,
                )?;
            }
```